assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -371,13 +476,23 @@ const MapDisplay = ({
       includeStopCard={true}
       stations={stations}
       shapes={routePatternShape ? [routePatternShape] : []}
+      // TODO: try rendering shapes as a child instead
```

Comment:
I want to avoid adding `onShapeSelect` as a base map prop. I think it would be ideal to just pass the `shapes` with their `onClick` as children to the `BaseMap`. @firestack I can't find the thread, but I remember you previously advocating for an approach passing children to the `BaseMap` over adding props. 

I'm interested to try it at this point since it may provide other opportunities to simplify some of the `liveSelectedEntity` complexity in MapDisplay. However,  the `RouteStopMarkers` component is currently dependent on zoom level, and I'm concerned that it would conflict with your current stop size work.